adapters/beachfront/beachfront.go

```diff
@@ -103,12 +99,17 @@ type beachfrontResponseSlot struct {
 	Adm   string  `json:"adm"`
 }
 
-type beachfrontVideoBidExtension struct {
+type videoBidExtension struct {
 	Duration int `json:"duration"`
 }
 
 func (a *BeachfrontAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	beachfrontRequests, errs := preprocess(request)
+	var errs []error
+	beachfrontRequests := preprocess(request, &errs)
```

Comment:
Please consider adding an error check after the `preprocess` function call. Also, you don't need to pass in the `errs` slice to the function. You can have the function return an error slice and then inspect that after the function is called or append that to `errs` slice if you'd like to move on but return the errors towards the end.

Reply:
I prefer Mansi's suggestion. It also allows you to know if the call produced new errors, or if the errors in the slice are old from elsewhere. You don't currently need to check for that, but very useful if you have the potential for a fatal error that you want to abort processing, you just need to check the new errors rather than traversing the entire array to see if a new fatal error was generated.

Reply:
Here's what I've currently got locally: